app/src/main/java/com/nadosunbae_android/presentation/ui/sign/SignUpMajorInfoActivity.kt

```diff
@@ -162,7 +150,7 @@ class SignUpMajorInfoActivity :
 
     //제 1전공 진입시기 선택 바텀시트
     private fun firstMajorPeriod() {
-        //bottomSheetDialog.binding.tvBottomsheeetTitle.setText("본 전공 진입시기")
+        bottomSheetDialog.binding.tvBottomsheeetTitle.setText("본 전공 진입시기")
```

Comment:
코틀린에서는 프로퍼티가 getter와 setter를 대체할 수 있기 때문에

```suggestion
        bottomSheetDialog.binding.tvBottomsheeetTitle.text = "본 전공 진입시기"
```
이런식으로만 작성해도 될 것 같습니다~!!